concepts/functions/about.md

```diff
@@ -0,0 +1,153 @@
+# About
+
+A function allows to group code into a reusable unit.
+There are multiple ways to define functions in JavaScript.
+Here we will look at function declarations and function expressions.
```

Comment:
```suggestion
Here we will look at _function declarations_ and _function expressions_.
```

Maybe some emphasis so we know it's a "term" and not two times two word groups?

Reply:
fixed in all the files